awxkit/awxkit/cli/options.py

```diff
@@ -171,12 +171,13 @@ def json_or_yaml(v):
                     except Exception:
                         raise argparse.ArgumentTypeError("{} is not valid JSON or YAML".format(v))
 
-                for k, v in parsed.items():
-                    # add support for file reading at top-level JSON keys
-                    # (to make things like SSH key data easier to work with)
-                    if isinstance(v, six.text_type) and v.startswith('@'):
-                        path = os.path.expanduser(v[1:])
-                        parsed[k] = open(path).read()
+                if isinstance(parsed, dict):
```

Comment:
Technically, JSON can be a list (as is the case w/ `--credentials` specified at launch time).